container/Dockerfile.tensorrt_llm

```diff
@@ -59,10 +59,31 @@ RUN wget https://github.com/etcd-io/etcd/releases/download/$ETCD_VERSION/etcd-$E
     rm /tmp/etcd.tar.gz
 ENV PATH=/usr/local/bin/etcd/:$PATH
 
-# TODO: Try using uv to install tensorrtllm
-ARG TENSORRTLLM_PIP_WHEEL_PATH=""
-COPY ${TENSORRTLLM_PIP_WHEEL_PATH}/*.whl /tmp/
-RUN find /tmp -name "*.whl" -exec pip install {} +
+ARG HAS_TRTLLM_CONTEXT=0
+ARG TENSORRTLLM_PIP_WHEEL="tensorrt-llm"
+ARG TENSORRTLLM_INDEX_URL="https://pypi.python.org/simple"
+
+COPY --from=trtllm_wheel . /trtllm_wheel/
+
+# TODO: Currently, ABI compatibility issues with TRTLLM wheel and NGC PyTorch prevent us
+# from using the TRTLLM wheel in a uv venv. Once the issues are resolved, we can
```

Comment:
Can you help me understand the relation to `uv` specifically here? What's different about `uv pip install` vs. `pip install` in terms of TRTLLM wheel ABI?

Reply:
I couldn't get the ABI compliance working in venv environment. Technically, there would be a way by defining an exact compliant venv as host. But I didn't spend much time on it. 
It is in my TODO to revisit and fix it.